BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -126,16 +132,28 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
 
     // MARK: - Private methods
 
-    private func loadInitialThemeType() -> ThemeType {
-        if let privateModeIsOn = userDefaults.object(forKey: ThemeKeys.PrivateMode.isOn) as? NSNumber,
-           privateModeIsOn.boolValue == true {
-            return .privateMode
-        }
+    private func updateSavedTheme(to newTheme: ThemeType) {
+        // As the private theme is a special instance of a theme,
+        // we don't want to save it.
+        guard newTheme != .privateMode else { return }
+        userDefaults.set(newTheme.rawValue, forKey: ThemeKeys.themeName)
+    }
```

Comment:
what clarifies a theme to be saved? In this case, we only want system theme. Should we add a comment for future us to know what qualifies a theme to be saved?

Reply:
I'll clarify.